server/src/routes/api/v1/reviewRouter.js

```diff
@@ -0,0 +1,33 @@
+import express from "express"
+import objection from "objection"
+const { ValidationError } = objection
+
+import { Review } from "../../../models/index.js"
+import cleanUserInput from "../../../services/cleanUserInput.js"
+
+const reviewRouter = new express.Router()
```

Comment:
This should be a nested router `trailReviewsRouter` because reviews belong to a specific trail. We want to hit this POST endpoint through `api/v1/trails/:trailId/reviews`